extensions/wikia/PhalanxII/fallback/hooks/UserCookieBlock.class.php

```diff
@@ -18,7 +18,7 @@ class UserCookieBlock extends UserBlock {
 	 * @param User $user
 	 * @return array|bool|null
 	 */
-	public static function blockCheck(User $user) {
+	public static function blockCheck( User $user ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)